src/cdk-experimental/menu/menu-key-manager.spec.ts

```diff
@@ -0,0 +1,752 @@
+import {
+  Component,
+  EventEmitter,
+  ViewChildren,
+  QueryList,
+  ElementRef,
+  ViewChild,
+} from '@angular/core';
+import {async, TestBed, ComponentFixture, tick, fakeAsync} from '@angular/core/testing';
+import {
+  dispatchKeyboardEvent,
+  createKeyboardEvent,
+  dispatchEvent,
+} from '@angular/cdk/testing/private';
+import {
+  TAB,
+  RIGHT_ARROW,
+  LEFT_ARROW,
+  DOWN_ARROW,
+  UP_ARROW,
+  SPACE,
+  HOME,
+  END,
+  E,
+  D,
+  ESCAPE,
+  S,
+  H,
+} from '@angular/cdk/keycodes';
+import {CdkMenuModule} from './menu-module';
+import {CdkMenuItem} from './menu-item';
+import {CdkMenuBar} from './menu-bar';
+import {CdkMenu} from './menu';
+import {defaultTypeAheadDebounce} from './menu-key-manager';
+import {CdkMenuItemCheckbox} from './menu-item-checkbox';
+import {CdkMenuItemRadio} from './menu-item-radio';
+
+describe('Keyboard Manager', () => {
+  describe('(with ltr layout)', () => {
+    let fixture: ComponentFixture<MultiMenuWithSubmenu>;
+    let nativeMenuBar: HTMLElement;
+    let nativeMenus: HTMLElement[];
+    let menuBarNativeItems: HTMLButtonElement[];
+    let fileMenuNativeItems: HTMLButtonElement[];
+
+    const grabElementsForTesting = () => {
+      nativeMenuBar = fixture.componentInstance.nativeMenuBar.nativeElement;
+
+      nativeMenus = fixture.componentInstance.nativeMenus.map(e => e.nativeElement);
+
+      menuBarNativeItems = fixture.componentInstance.nativeItems
+        .map(e => e.nativeElement)
+        .slice(0, 2);
+
+      if (nativeMenus.length > 0) {
+        fileMenuNativeItems = fixture.componentInstance.nativeItems
+          .map(e => e.nativeElement)
+          .slice(2, 5);
+      }
+    };
+
+    /** Run change detection and extract then set the rendered elements. */
+    const detectChanges = () => {
```

Comment:
For standalone functions like this, we typically use the `function` keyword
```suggestion
function detectChanges() {
}
```